tests/integration/programs.test.js

```diff
@@ -35,15 +36,27 @@ describe('PROGRAMS ROUTES - POST /programs', () => {
     });
 
     it('should create program', async () => {
```

Comment:
ici tu peux faire 2 tests distincts `should create program name` et `should create program categories`